inference-engine/src/transformations/src/transformations/mish_fusion.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/mish_fusion.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+ngraph::pass::MishFusion::MishFusion() {
```

Comment:
This transformation should be included into some list of transformation. Probably to `inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp`.

@GlebKazantaev, what do you think?